CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 6.0.0 - 2022-05-02
+### Removed
+- Removed support for PHP 7.1, 7.2 and 7.3
+- Removed support for Symfony 3.4
+- Removed unused `z2.yml` and `phpunit.xml.dist` files
+### Fixed
+- Changed use of deprecated Symfony FrameworkBundle Controller class into AbstractController class and
+  inject the Solr Client into the constructor
+- Pass root name to TreeBuilder constructor and call `getRootNode` instead of deprecated `->root()`
+### Added
+- Added autotagging of the `zicht_solr.mapper` tag for classes extending the `EntityMapper` class. So no
+  _instanceof EntityMapper > tag zicht_solr.mapper_ like configuration is needed in the projects itself.
```

Comment:
Hoe zag dat er uit in projecten met SOLR-bundle v5 en lager? Wat moesten we hier precies voor doen? (En waarom?)

Reply:
Traditioneel (pre-Sf4) voegden we een service definitie toe voor elke service en dus ook voor Solr mappers. Bij solr mappers moest je dan een `zicht_solr.mapper` tag toevoegen.

Met autowiring en autoconfiguration mogelijkheden deden we nu in Sf 4 projecten:

```yaml
services:
    _instanceof:
        Zicht\Bundle\SolrBundle\Manager\Doctrine\EntityMapper:
            autowire: true
            tags: ['zicht_solr.mapper']
```

Nu heb ik in de bundle zelf dus gezegd: als iets een instance is van `EntityMapper`, dan krijgt deze de tag `zicht_solr.mapper`, dus hoef je in projecten niks meer te doen hiervoor. Symfony doet dit zelf ook met allerlei interfaces (EventSubscriberInterface, Twig\Extension\ExtensionInterface etc.).

Reply:
Ahh ik denk dat ik die syntax nooit meer heb gebruikt omdat ik geen nieuwe SF4 projecten met SOLR meer ben gestart.